storage/rocksdb/CMakeLists.txt

```diff
@@ -82,14 +101,10 @@ ENDIF()
 
 SET(rocksdb_static_libs ${rocksdb_static_libs} ${ZLIB_LIBRARY} "-lrt")
 
-MYSQL_ADD_PLUGIN(rocksdb_se ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT STATIC_ONLY
+MYSQL_ADD_PLUGIN(rocksdb ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT MODULE_ONLY
   LINK_LIBRARIES ${rocksdb_static_libs}
 )
 
-IF(WITH_EMBEDDED_SERVER)
```

Comment:
Proactive, I can't recall ever even trying to build an embedded Percona Server as it was always my understanding that it just didn't work. I can pull it if you feel strongly about it but embedded is something that is just not at all on my radar here.